cypress/support/fragments/settings/finance/settingsFinance.js

```diff
@@ -0,0 +1,73 @@
+import { Button, TextField, MultiColumnListHeader, EditableListRow, MultiColumnListCell } from '../../../../../interactors';
+import InteractorsTools from '../../../utils/interactorsTools';
+import DateTools from '../../../utils/dateTools';
+
+const saveButton = Button('Save');
+const deleteButton = Button('Delete');
+const trashIconButton = Button({ icon: 'trash' });
+const editIconButton = Button({ icon: 'edit' });
+function getEditableListRow(rowNumber) { return EditableListRow({ index: +rowNumber.split('-')[1] }); }
+
+export default {
+
+  waitExpenseClassesLoading: () => {
+    cy.expect(MultiColumnListHeader({ id: 'list-column-name' }).exists());
+  },
+
+  fillRequiredFields: (expenseClasses) => {
+    cy.do([
+      TextField({ placeholder: 'name' }).fillIn(expenseClasses.name),
+      TextField({ placeholder: 'code' }).fillIn(expenseClasses.code),
+      TextField({ placeholder: 'externalAccountNumberExt' }).fillIn(expenseClasses.externalAccountNumber),
+      saveButton.click()
+    ]);
+  },
+
+  createNewExpenseClass(expenseClass) {
+    cy.do(Button({ id: 'clickable-add-expenseClasses' }).click());
+    this.fillRequiredFields(expenseClass);
+  },
+
+  editExpenseClass(expenseClass, oldExpenseClassName) {
+    cy.do(MultiColumnListCell({ content: oldExpenseClassName }).perform(
+      element => {
+        const rowNumber = element.parentElement.parentElement.getAttribute('data-row-index');
+        cy.do(getEditableListRow(rowNumber).find(editIconButton).click());
+        this.fillRequiredFields(expenseClass);
+      }
+    ));
+  },
+
+  checkExpenseClass: (expenseClass) => {
+    cy.do(MultiColumnListCell({ content: expenseClass.name }).perform(
+      element => {
+        const rowNumber = element.parentElement.parentElement.getAttribute('data-row-index');
+        const createdByAdmin = `${DateTools.getFormattedDateWithSlashes({ date: new Date() })} by ADMINISTRATOR, DIKU`;
```

Comment:
please think about the case when the firstname and lastname of the admin user have other values